plugins/web-worker-offloading/third-party/seo-by-rank-math.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Web Worker Offloading integration with Rank Math SEO.
+ *
+ * @since n.e.x.t
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Configures WWO for Rank Math SEO and Google Analytics.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @link https://partytown.builder.io/google-tag-manager#forward-events
+ *
+ * @param array<string, mixed>|mixed $configuration Configuration.
+ * @return array<string, mixed> Configuration.
+ */
+function plwwo_rank_math_configure( $configuration ): array {
+	$configuration = (array) $configuration;
+
+	$configuration['globalFns'][] = 'gtag'; // Because gtag() is defined in one script and called in another.
+	$configuration['forward'][]   = 'dataLayer.push'; // Because the Partytown integration has this in its example config.
+	return $configuration;
+}
+add_filter( 'plwwo_configuration', 'plwwo_rank_math_configure' );
+
+/*
+ * Note: The following integration is not targeting the \RankMath\Analytics\GTag::enqueue_gtag_js() code which is only
+ * used for WP<5.7. In WP 5.7, the wp_script_attributes and wp_inline_script_attributes filters were introduced, and
+ * Rank Math then deemed it preferable to use wp_print_script_tag() and wp_print_inline_script_tag() rather than
+ * wp_enqueue_script() and wp_add_inline_script(), respectively. Since Web Worker Offloading requires WP 6.5+, there
+ * is no point to integrate with the pre-5.7 code in Rank Math.
+ */
+
+/**
+ * Filters script attributes to offload Rank Math's GTag script tag to Partytown.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @link https://github.com/rankmath/seo-by-rank-math/blob/c78adba6f78079f27ff1430fabb75c6ac3916240/includes/modules/analytics/class-gtag.php#L161-L167
```

Comment:
assume this is a "permalink"?

Reply:
It's a [`@link`](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.link.pkg.html) to show what part of RankMath this code is related to.

Reply:
right, I meant the github link is from the 'copy permalink' menu:
![image](https://github.com/user-attachments/assets/b3af4b8b-83aa-4f6f-b0a9-52aafdf76099)

I get https://github.com/rankmath/seo-by-rank-math/blob/6d081340b126ea80232b0bafa295ede6fa69430b/includes/modules/analytics/class-gtag.php#L161-L167 for that on master

Reply:
* I believe using this type of link makes it work even if the code is moved or deleted.

Reply:
Oh yes, sorry, exactly. I forgot that GitHub used the "permalink" term. So yes, this link ensures that the code reference doesn't change when changes are added.